spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -751,6 +753,129 @@ definitions:
+       - update_interval:
+           type: u8
+           desc: >
+             Update interval between consecutive corrections. Encoded
+             following RTCM DF391 specification.
+       - sol_id:
+           type: u8
+           desc: SSR Solution ID. Similar to RTCM DF415.
+       - iod_ssr:
+           type: u8
+           desc: >
+             IOD of the SSR correction. A change of Issue Of Data
+             SSR is used to indicate a change in the SSR
+             generating configuration
+       - tile_set_id:
+           type: u16
+           desc: Unique identifier of the tile set this tile belongs to.
+       - tile_id:
+           type: u16
+           desc: >
+             Unique identifier of this tile in the tile set.
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field correctionPointSetID.
+       - corner_nw_lat:
+           type: s16
+           units: encoded degrees
+           desc: >
+             North-West corner correction point latitude.
+             
+             
+             The relation between the latitude X in the range [-90, 90] and
+             the coded number N is:
+             
+             
+             N = floor((X / 90) * 2^14)
+             
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field referencePointLatitude.
+       - corner_nw_lon:
+           type: s16
+           units: encoded degrees
+           desc: >
+             North-West corner correction point longitude.
+             
+             
+             The relation between the longitude X in the range [-180, 180]
+             and the coded number N is:
+             
+             
+             N = floor((X / 180) * 2^15)
+             
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field referencePointLongitude.
+       - spacing_lat:
+           type: u16
+           units: 0.01 degrees
+           desc: >
+             Spacing of the correction points in the latitude direction.
+             
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field stepOfLatitude.
+       - spacing_lon:
+           type: u16
+           units: 0.01 degrees
+           desc: >
+             Spacing of the correction points in the longitude direction.
+             
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field stepOfLongitude.
+       - rows:
+           type: u16
+           desc: >
+             Number of steps in the latitude direction.
+             
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field numberOfStepsLatitude.
+       - cols:
+           type: u16
+           desc: >
+             Number of steps in the longitude direction.
+             
+             
+             See GNSS-SSR-ArrayOfCorrectionPoints field numberOfStepsLongitude.
+       - bitmask:
+           type: u64
+           desc: >
+             Specifies the availability of correction data at the
```

Comment:
Need to see if we can make this documentation more succinct without reducing clarity to make things a little easier on the PDF, right now we're overflowing by a few lines on the end of the table describing these fields:

![image](https://user-images.githubusercontent.com/183436/211942844-7b9f8767-0065-4cb6-a466-c514e097c927.png)